tikv-client/src/main/java/com/pingcap/tikv/region/RegionStoreClient.java

```diff
@@ -122,6 +123,24 @@ public ByteString get(BackOffer backOffer, ByteString key, long version)
     }
   }
 
+  private boolean checkLockError(BackOffer backOffer, Kvrpcpb.KeyError error) {
+    if (error.hasLocked()) {
+      Lock lock = new Lock(error.getLocked());
+      boolean ok =
+          lockResolverClient.resolveLocks(
+              backOffer, new ArrayList<>(Collections.singletonList(lock)));
+      if (!ok) {
+        // if not resolve all locks, we wait and retry
```

Comment:
`all locks`?  maybe there's only one lock

Reply:
Maybe it is better to change to `if any lock unresolved`